### ISSUE TITLE
State of JS 2024 Suggestions

### ISSUE THREAD

**Issue by SachaG** [Devographics/surveys#227]
Add suggestions here.

**Comment by SachaG** [comment]
> Is it possible to get [@eleventy](https://hachyderm.io/@eleventy@fosstodon.org) removed from the State of JavaScript survey?

**Comment by SachaG** [comment]
https://www.rspack.dev/

**Comment by SachaG** [comment]
https://rive.app/

**Comment by Schepp** [comment]
While it is already part of the State of CSS Survey (thanks for that!), I'd suggest to also include the [Working Draft podcast](https://workingdraft.de) into the JS and HTML editions as well, as we evenly pick and discuss topics from all parts of the web platform. In 2023 we had a bunch of mixed episodes where we explain new or randomly picked features of TypeScript, EcmaScript or the DOM API, but also a few episodes dedicated to JS concepts or frameworks or platforms like Signals, Qwik, Solid & Solidstart, SvelteKit, and CloudFlare Pages: https://workingdraft.de/archiv/

**Comment by SachaG** [comment]
@Schepp thanks for the suggestion! This year (2023) we actually went with an open-ended format for the resources questions so we'll probably base the 2024 list based on the top answers to that.

**Comment by SachaG** [comment]
https://nitro.unjs.io/
https://analogjs.org/
https://vinxi.vercel.app/

**Comment by nzakas** [comment]
Linter category:

* [ESLint](eslint.org)
* [Deno Lint](https://docs.deno.com/runtime/manual/tools/linter)
* [Biome](https://biomejs.dev/linter/)

**Comment by AugustinMauroy** [comment]
On test category:
* node test runner: https://nodejs.org/en/learn/test-runner/introduction https://nodejs.org/docs/latest/api/test.html#test-runner

On the subject of monorepo tools, you should separate the package managers that support monorepo and the help/caching tools ... 

We have "JavaScript/TypeScript Balance", it would be nice to have something about ESM/CJS.


There should also be a question about JSR or npm registries.

**Comment by SachaG** [comment]
Rspack
Rolldown

**Comment by forgotPassword** [comment]
Regarding `Missing Features` question, how does one vote "stop fucking up js, don't add this"? For example "static typing" is "the most requested feature" in past years, yet it means nothing since there is no option to vote for people who do not want this feature.

**Comment by SachaG** [comment]
> Regarding `Missing Features` question, how does one vote "stop fucking up js, don't add this"? For example "static typing" is "the most requested feature" in past years, yet it means nothing since there is no option to vote for people who do not want this feature.

That's like saying that asking people their favorite color is meaningless unless you also ask them what color they hate the most. They're two different datapoints and they're both interesting, but I think it's OK to ask about one and not the other.

**Comment by forgotPassword** [comment]
I think explicitly not wanting static typing is a valid opinion, and right now there is no way to vote for that. Maybe each option could have total up-votes and down-votes.

**Comment by SachaG** [comment]
I agree it's a valid question, it's just not one we're planning to ask currently. 

An upvote/downvote system would be interesting, but the risk would be that most people would just upvote everything unless we put limits on how many items you can upvote/downvote. It's also a UX challenge to keep things clear/simple/etc.
